flask_app/utils.py

```diff
@@ -0,0 +1,73 @@
+import functools
```

Comment:
This "utils" file is 95% `RemoteHost` class, and 5% another function.
I suggest:
- Have a `utils` folder
- Create a module  `remote_host.py`
- Put `combadge.py` there, which is better to rename to `remote_combadge.py`.